src/io/flutter/perf/EditorPerfDecorations.java

```diff
@@ -435,13 +448,14 @@ else if (stats.getKind() == PerfReportKind.repaint) {
       sb.append(" counts for: <strong>" + stats.getDescription());
       sb.append("</strong></p>");
       sb.append("<p style='padding-left: 8px'>");
-      sb.append("In past second: " + stats.getPastSecond() + "<br>");
-      sb.append("Since last route change: " + stats.getTotalSinceNavigation() + "<br>");
-      sb.append("Since last hot reload/restart: " + stats.getTotal());
+      sb.append("For last frame: " + stats.getValue(PerfMetric.lastFrame) + "<br>");
+      sb.append("In past second: " + stats.getValue(PerfMetric.pastSecond) + "<br>");
+      sb.append("Since last route change: " + stats.getValue(PerfMetric.totalSinceRouteChange) + "<br>");
+      sb.append("Since last hot reload/restart: " + stats.getValue(PerfMetric.total));
       sb.append("</p>");
     }
     if (sb.length() == 0) {
-      sb.append("<p><b>No widget rebuilds or repaints detected for line.</p></b>");
+      sb.append("<p><b>No widget rebuilds detected for line.</p></b>");
```

Comment:
`End with </b></p> since it starts with <p><b>`

Reply:
done